### ISSUE TITLE
About diversity loss function in your paper

### ISSUE THREAD

**Issue by asama-yaya** [ChenyangLEI/Fully-Automatic-Video-Colorization-with-Self-Regularization-and-Diversity#1]
Hi, Mr.Chenyang and Mr.Qifeng.
I have a question about in your paper "Fully-Automatic-Video-Colorization-with-Self-Regularization-and-Diversity".

In your paper, you used φ in your proposed loss function for diversity.
I wonder what this φ represents.

I checked your cited paper "Interactive_Image_Segmentation_with_Latent_Diversity", but I can't find φ.
Could you tell me, what do φ represents?

Thank you.

**Comment by ChenyangLEI** [comment]
Hi, 
φ mean the feature maps of pretrained VGG-19 network. In practice, we use "conv1_2", "conv2_2", "conv3_2", "conv4_2", "conv5_2" layers.

**Comment by asama-yaya** [comment]
Thank you!
